src/components/Statuses/Statuses.module.scss

```diff
@@ -1,10 +1,23 @@
 .base {
+  padding: 0 1rem;
+  &:not(:first-of-type) {
+    margin-top: 1rem;
+  }
+}
```

Comment:
The component bases are not meant to exist multiple times. Please use a different element to wrap each group.

Reply:
Wasn't sure about this one, as on a previous test, only the padding was left in this class. But sure can rewind to the use of a group class 👍